src/umpire/resource/NoOpResourceFactory.cpp

```diff
@@ -0,0 +1,58 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include "umpire/resource/NoOpResourceFactory.hpp"
+#include "umpire/resource/NoOpMemoryResource.hpp"
+
+#include "umpire/util/Macros.hpp"
+#include "umpire/util/make_unique.hpp"
+#include "umpire/util/detect_vendor.hpp"
+
+namespace umpire {
+namespace resource {
+
+bool NoOpResourceFactory::isValidMemoryResourceFor(
+    const std::string& name) noexcept
+{
+  if (name.find("No_Op") == std::string::npos)
```

Comment:
I have seen (I think) "NO_OP" and "No_Op", I think it needs to be one or the other.  I suggest either "NO_OP" or "NOOP".

Reply:
I tried to use "NO_OP" but it gave me an exception with "getDefaultTraitsForResource MemoryResource NO_OP not found" - so I switched it back lol i thought that would have worked (?)

Reply:
```suggestion
  if (name.find("NO_OP") != std::string::npos)
```

Reply:
JK, fixing it now!